test/test_cuda.py

```diff
@@ -4702,6 +4703,98 @@ class TestCudaOptims(TestCase):
     # These tests will be instantiate with instantiate_device_type_tests
     # to apply the new OptimizerInfo structure.
 
+    @onlyCUDA
+    @unittest.skipIf(
+        not TEST_CUDA_GRAPH, "CUDA >= 11.0 or ROCM >= 5.3 required for graphs"
+    )
+    @parametrize("foreach, fused", [(False, False), (True, False), (False, True)])
```

Comment:
why is this necessary?

Reply:
@janeyx99 I have meta devgpu, it looks like the supports_fused_on didn't had "cuda:0" entry. 

all 4 fused tests are passing (4 fused entries in [optim_db](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_optimizers.py#L1150) )